<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialInfo</class>
 <widget class="QWidget" name="TutorialInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TutorialInfo {
background-color: rgba(20,20,20,200);
}
#pushButton {
border-style: solid;
font-size: 36px;
background-color: rgba(30,30,30,150);
color: rgb(180,180,180);
border-radius: 20px;
}
#pushButton:hover {
background-color: rgba(20,20,20,255);
border-color: rgb(180,180,180);
border-width: 2px;
}
#firstLabel {
color: rgb(180,180,180);
font-size: 32px;
}
#secondLabel {
color: rgb(180,180,180);
font-size: 28px;
}
#attackActionLabel, #defenseActionLabel, #healActionLabel {
color: rgb(180,180,180);
font-size: 20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,3,2,2,2,1,1">
   <item>
    <widget class="QLabel" name="firstLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Włoszczowa nie jest bezpiecznym miejscem!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="secondLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Podczas swojej podróży napotkasz wielu niebezpiecznych przeciwników, którzy rzucą ci wyzwanie.&lt;/p&gt;&lt;p&gt;W swojej turze możesz wybrać 1 z 3 akcji:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,1">
     <item>
      <widget class="QLabel" name="attackActionIconLabel">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/images/AppScreenArt/DarkSword.png) 0 0 0 0 stretch stretch;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="attackActionLabel">
       <property name="text">
        <string>atak - zadaje przeciwnikowi obrażenia bazując na twoich punktach ataku</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1">
     <item>
      <widget class="QLabel" name="defenseActionIconLabel">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/images/AppScreenArt/DarkShield.png) 0 0 0 0 stretch stretch;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="defenseActionLabel">
       <property name="text">
        <string>obrona - zmniejsza otrzymywane w następnej turze obrażenia i zapewnia szansę na odbicie części obrażeń w stronę wroga</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1">
     <item>
      <widget class="QLabel" name="healActionIconLabel">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/images/AppScreenArt/DarkHeart.png) 0 0 0 0 stretch stretch;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="healActionLabel">
       <property name="text">
        <string>leczenie - odzyskujesz część utraconego przez ciebie życia</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
